crates/anvil/src/cmd.rs

```diff
@@ -218,9 +219,11 @@ impl NodeArgs {
         let hardfork = match &self.hardfork {
             Some(hf) => {
                 if self.evm.optimism {
-                    Some(OptimismHardfork::from_str(hf)?.into())
+                    let hf = OpHardfork::from_str(hf).map_err(|s| eyre::eyre!(s))?.into();
```

Comment:
@Rimeeeeee I feel like this impl should return an actual error,

can we update this on alloy hards, then we dont need to change anything here